lib/CXGN/Stock.pm

```diff
@@ -282,9 +282,9 @@ sub store {
                 $self->_store_stockprop('organization', $self->organization_name());
             }
             if ($self->population_name){
-		print STDERR "**STOCK.PM This stock has population name " . $self->population_name . "\n\n";
-		#DO NOT INSERT POPULATION RELATIONSHIP FROM THE STOCK STORE FUNCTION
-		##$self->_store_population_relationship();
+                print STDERR "**STOCK.PM This stock has population name " . $self->population_name . "\n\n";
+                #DO NOT INSERT POPULATION RELATIONSHIP FROM THE STOCK STORE FUNCTION
+                $self->_store_population_relationship();
```

Comment:
I think we need to remove this from the store function and explicitly call store_population_relationship from the AddPopulation object. This side effect can store unwanted stocks and stock_relationships unintentionally